yield_plugin/test/test_yield_plugin.cpp

```diff
@@ -43,12 +43,16 @@ class YieldNode :  public carma_ros2_utils::CarmaLifecycleNode
     {
       RCLCPP_INFO(get_logger(),"Test lc callback..");
     }
-    
+
 };
 
+// These tests has been temporarily disabled to support Continuous Improvement (CI) processes.
+// Related GitHub Issue: <https://github.com/usdot-fhwa-stol/carma-platform/issues/2335>
+
+/**
```

Comment:
@MishkaMN @adamlm do we want to prioritize fixing these unit tests as part of the release for VRU?